### PR TITLE
Modify OWNERS to define labels

### LINKED ISSUE #108: RFE: Add kind/blog tag for PR's that contain _posts/ in path
Would be good to have the PR's tagged with 'kind/blog' for PR's with files in the `_post/` path

### PR DESCRIPTION
**Pull request by iranzo**
Closes #108

### CODE REVIEW

**Comment by iranzo** [comment]
@russellb something that I should fix here?

File: OWNERS [hunk 1]
```diff
@@ -1,11 +1,23 @@
 # See the OWNERS docs at https://go.k8s.io/owners
+filters:
+  ".*":
+    reviewers:
```
Comment (russellb) [inline OWNERS]: did you mean to change half of this group to reviewers instead of approvers?

Comment (iranzo) [reply OWNERS]: Well, as it was said to have you to review them that was the goal. Would you suggest to adapt it in other way?

Comment (russellb) [reply OWNERS]: just the blogs / technical content - i don't want to be in the critical path for everything :)

Comment (iranzo) [reply OWNERS]: Tried to adjust it, let's see if it works as expected :-)

Comment (russellb) [reply OWNERS]: I don't think it hurts anything, but I believe listing people as reviewers is redundant if they're already listed as approvers.  I don't think being a reviewer gives you anything more than what you get by default as a member of the github org with our setup.

**Pull request merged**